okapi-core/src/test/java/org/folio/okapi/util/ModuleUtilTest.java

```diff
@@ -266,5 +331,33 @@ void testGetParamInteger() {
     assertThrows(DecodeException.class, () -> ModuleUtil.getParamInteger(params, "y", 1));
   }
 
-
+  @Test
+  void testTnterfaceCheckProvide() {
+    assertThat(ModuleUtil.interfaceCheck(null, "int", null)).isFalse();
+    InterfaceDescriptor [] interfaces = new InterfaceDescriptor[] {
+        new InterfaceDescriptor("int", "1.1 2.0")
+    };
+    assertThat(ModuleUtil.interfaceCheck(interfaces, null, null)).isTrue();
+    assertThat(ModuleUtil.interfaceCheck(interfaces, "int", null)).isTrue();
+    assertThat(ModuleUtil.interfaceCheck(interfaces, "no", null)).isFalse();
+    assertThat(ModuleUtil.interfaceCheck(interfaces, "int,no", null)).isTrue();
+    assertThat(ModuleUtil.interfaceCheck(interfaces, "no,int", null)).isTrue();
+    assertThat(ModuleUtil.interfaceCheck(interfaces, "int=0.9", null)).isFalse();
+    assertThat(ModuleUtil.interfaceCheck(interfaces, "int=1.1", null)).isTrue();
+    assertThat(ModuleUtil.interfaceCheck(interfaces, "int=1.0", null)).isFalse();
+    assertThat(ModuleUtil.interfaceCheck(interfaces, "int=1.2", null)).isFalse();
```

Comment:
Why is this false, 1.2 is compatible with the required 1.1?

Reply:
I decided to go for exact match.